Templates/GitHubActionsPipeline/.github/workflows/pipelines/Build.yml

```diff
@@ -0,0 +1,109 @@
+name: Build
+on:
+    workflow_call:
+        secrets:
+            githubAccessToken:
+                required: true
+        inputs:
+            pipelineType:
+                description: Pipeline Type
+                default: build
+                type: string
+                required: false
+
+env:
+    uniqueWorkspaceId: /u/github/workspace/${{ github.repository }}/${{ github.ref_name }}/build_b${{github.run_number }} # adding a b to before the number,
+    # to avoid an error caused by repeated build numbers 
+    branch : ${{ github.ref_name }} 
+    application: ${{ github.event.repository.name }}
+    pipelineType: ${{ inputs.pipelineType }}
+    packageName: ${{github.run_id}}
+    zosHostname: lpar.dev.ibm.com
+    zosSFTPUser: GITHUB@lpar.dev.ibm.com
+
+jobs:
+    Setup:
+        runs-on: self-hosted
+        steps:
+         - name: CI-CD pipeline parameter printout
+           run: |
+            echo " "
+            echo "Following are the CI-CD pipeline parameters"
+            echo "*******************************************"
+            echo "uniqueWorkspaceId                  = ${{ env.uniqueWorkspaceId}}"
+            echo "branch                             = ${{ env.branch}}"
+            echo "application                        = ${{ env.application}}"
+            echo "pipelineType                       = ${{ env.pipelineType}}"
+            echo "packageName                        = ${{ env.packageName}}"
+            echo "zosHostname                        = ${{ env.zosHostname}}"
+            echo "zosSFTPUser                        = ${{ env.zosSFTPUser}}"
+         - name: Clone Repo to z/OS Unix System Services
+           run: ssh ${{ env.zosSFTPUser}} ". ./.profile && gitClone.sh -w ${{ env.uniqueWorkspaceId }} -r https://${{ secrets.githubAccessToken }}@github.com/${{ github.repository }}.git -b ${{ env.branch }}" 
+
+    Build:
+        needs: Setup
+        runs-on: self-hosted
+        steps:
+          - name: Build git repo
+            run: ssh ${{ env.zosSFTPUser}} ". ./.profile && dbbBuild.sh -w ${{ env.uniqueWorkspaceId }} -a ${{env.application}} -b ${{ env.branch }} -p ${{env.pipelineType}} -t '--fullBuild'"
```

Comment:
`-t '--fullBuild'` is an override of the build type, please remove this from the template (might be on other workflow files as well?